playbooks/roles/nginx_vhosts/templates/fragments/vhost_standalone.conf

```diff
@@ -3,6 +3,12 @@
 	autoindex on;	
 	try_files $uri $uri/ {{item.alt_404|default("=404")}};
 	
+	{%if item.is_static_site is defined %}
```

Comment:
Instead of doing this, could we just set `alt_404` to `index.html`? Seems to me that would have the same effect without needing to add another flag.